### ISSUE TITLE
Could you tell me what masklist/train2017.txt ?

### ISSUE THREAD

**Issue by tugumi911** [last-one/Pytorch_Realtime_Multi-Person_Pose_Estimation#6]
Sorry
I run preproccesing/generate_json_mask.py, but masklist/train2017.txt was not generated.
What is the file ?

And I have no multi gpu but one gpu, should I remove line 49 "model = torch.nn.DataParallel(model, device_ids=args.gpu).cuda()" in train_pose.py ?

**Comment by last-one** [comment]
The mask has been generated. You can generate the file by youself.

And you can remove this line.

**Comment by tugumi911** [comment]
Thank you for replying !
Is mask_dir masklist/train2014.txt's path ?

**Comment by tztztztztz** [comment]
I have the same question. I just got a directory which contains a lot of .npy file,  rather than a single .txt mask file? @last-one

**Comment by last-one** [comment]
Yes. It's the mask_dir

**Comment by last-one** [comment]
It needs to be generated by yourself.

**Comment by tztztztztz** [comment]
@last-one thanks for your reply. fixed it

**Comment by wangfei-cs** [comment]
@last-one 
You said 'The mask has been generated. You can generate the file by youself.'
sorry, but how do i generate the file by myself?

**Comment by last-one** [comment]
@wangfei-cs please see README

**Comment by last-one** [comment]
I have added the code for generating masklist/train2017.txt. Please see the script preprocessing/generate_json_mask.py
